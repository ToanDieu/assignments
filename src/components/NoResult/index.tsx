import React from "react";
import { Container, Row, Col } from "react-bootstrap";

import "./index.scss";

export default function NoResult() {
  return (
    <Container className={"list-forecast"}>
      <Row>
        <Col sm={12} md={12} lg={12}>
          <h1 className={"text-center"}>No result</h1>
        </Col>
      </Row>
    </Container>
  );
}
