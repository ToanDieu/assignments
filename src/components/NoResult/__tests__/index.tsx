import { render } from "@testing-library/react";

import NoResult from "../index";

describe("<NoResult/>", () => {
  test("render No Result text", () => {
    const { container } = render(<NoResult />);
    const h1 = container.querySelector("h1");
    expect(h1?.textContent).toBe("No result");
  });
  test("NoResult should render and match snapshort", () => {
    const { container } = render(<NoResult />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
