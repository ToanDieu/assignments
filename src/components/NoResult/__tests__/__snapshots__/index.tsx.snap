// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResult/> NoResult should render and match snapshort 1`] = `
<div
  class="list-forecast container"
>
  <div
    class="row"
  >
    <div
      class="col-lg-12 col-md-12 col-sm-12"
    >
      <h1
        class="text-center"
      >
        No result
      </h1>
    </div>
  </div>
</div>
`;
